test/integration/golden_response/staging/get_stat_ranking/country.json

```diff
@@ -5,12 +5,12 @@
         "info": [
           {
             "rank": 1,
-            "value": 1397715000,
+            "value": 1396530000,
```

Comment:
I suspect this is because of OECD update.  @hanlu09205 

We can check after push to staging.